### PR TITLE
Updates for 0.12.5

### LINKED ISSUE #103: orphan 'tutorials' link visible in TOC
At some point recently, the orphan 'tutorials' link snuck back into the TOC:

| latest | 0.11 |
|--|--|
|image | image |

refs:

* https://github.com/learningequality/kolibri-docs/pull/52
* https://github.com/learningequality/kolibri-docs/issues/60

### PR DESCRIPTION
**Pull request by radinamatic**
- [X] Added steps to include Kolibri to [antivirus exclusion list](https://learningequality.slack.com/archives/C0LKF6TRD/p1560443731004000) on Windows.
- [X] Clarify that [coach resources included in lessons and quizzes *are* visible to learners](https://learningequality.slack.com/archives/CB37UM23A/p1560992040244800).
- [X] Exclude the *Tutorials* section from the TOC. Fixes #103.
- [ ] Tried to fix the warning while viewing documentation for an older release, but couldn't make it work. I'll continue on it in a separate PR so this can be merged.

### CODE REVIEW

File: docs/manage/_get_support.rst [hunk 1]
```diff
@@ -52,6 +52,20 @@ Videos are not playing
 Make sure to check the :ref:`system requirements <sys_reqs>` to see if you can support video playback. Please report any issues on our `Community Forums <https://community.learningequality.org/>`_, stating the operating system and browser you are using.
 
 
+Antivirus
+---------
+
+Some overzealous antivirus programs on Windows platform may preventively impede Kolibri or some of its components (for example ``python.exe``) from running correctly. If that happens you need to add them to the antivirus exclusion list. Below steps refer to the program **Avast**, but should be similar in other antivirus applications.
```
Comment (benjaoming) [inline docs/manage/_get_support.rst:58]: I'm a fan :heart:

**Review by benjaoming — APPROVED** [review]
Wow, some really nice typo and spelling catches in this one! LGTM

**Comment by benjaoming** [comment]
> Tried to fix the warning while viewing documentation for an older release, but couldn't make it work. I'll continue on it in a separate PR so this can be merged.

That's fair.. I also tried to make it work by doing some rebuilds. Ultimately, I would like to have tried publishing tagged builds to see if this would trigger something.. but somehow the panel for selecting branches and tags is missing from current ReadTheDocs interface.

File: docs/manage/_get_support.rst (hunk 1 above)
Comment (radinamatic) [reply docs/manage/_get_support.rst:58]: I can't actually remember the last time I had antivirus on any of my devices... Must be at least 15 years :scream: :sunglasses:

**Comment by radinamatic** [comment]
Thanks for the review @benjaoming, merging!

**Pull request merged**